package.json

```diff
@@ -9,12 +9,12 @@
   },
   "devDependencies": {
     "chai": "3.5.0",
-    "coffee-script": "1.12.6",
     "coveralls": "2.13.1",
     "css-parse": "2.0.0",
     "istanbul": "0.4.5",
     "mocha": "3.4.1",
     "mocha-lcov-reporter": "1.3.0",
+    "prettier": "^1.13.7",
```

Comment:
Prettier is a dep but is not actually used anywhere (other than perhaps your personal vscode settings). If you'd like for the project to use prettier for formatting, it needs to be implemented somehow. Typically I use a "format" npm script set up with a pre-commit hook.